tests/default/_core/aggregation/nested.yaml

```diff
@@ -0,0 +1,100 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
```

Comment:
Let's map the file name to the query, so this should be `search/aggs/nested_agg.yaml`.